specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/managedInstanceSensitivityLabels.json

```diff
@@ -0,0 +1,523 @@
+        "operationId": "ManagedInstanceSensitivityLabels_ListByDatabase",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "An OData filter expression that filters elements in the collection.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the sensitivity labels.",
+            "schema": {
+              "$ref": "#/definitions/SensitivityLabelListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 SensitivityLabelsInvalidODataQuery - The specified OData query is either not valid or not supported\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 DatabaseDoesNotExist - User has specified a database name that does not exist on this server instance."
+          }
+        },
+        "x-ms-odata": "#/definitions/SensitivityLabel",
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets the current and recommended sensitivity labels of a given database": {
+            "$ref": "./examples/ManagedInstanceSensitivityLabelsListByDatabase.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/databases/{databaseName}/sensitivityLabels/{sensitivityLabelSource}": {
+      "get": {
+        "tags": [
+          "ManagedInstanceSensitivityLabels"
+        ],
+        "description": "Gets the sensitivity labels of a given database",
+        "operationId": "ManagedInstanceSensitivityLabels_ListByDatabaseWithSource",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "sensitivityLabelSource",
+            "in": "path",
+            "description": "Optional source of the sensitivity label. Valid values are current or recommeneded. In not specified both returned.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "current",
+              "recommended"
```

Comment:
Just to confirm the behavior:
For the GET and LIST operations, this can have a value of "current" or "recommended"
For the PUT and DELETE operations, this can *only* have a value of "current"

Is that correct?

It's worth noting that required enums of a single value (like "current") are treated as a constant in the generated SDKs and are therefore not exposed to the user and are simply added by default in the client.

Reply:
All correct beside LIST. There the API is 
../databases/{databaseName}/sensitivityLabels
../databases/{databaseName}/currentSensitivityLabels
../databases/{databaseName}/recommendedSensitivityLabels

the sensitivityLabelSource parameter is removed there